src/Compilers/Core/Portable/Operations/OperationNodes.cs

```diff
@@ -47,8 +47,8 @@ internal abstract class LazyNoneOperation : BaseNoneOperation
     {
         private ImmutableArray<IOperation> _lazyChildrenInterlocked;
 
-        public LazyNoneOperation(SemanticModel semanticModel, SyntaxNode node, Optional<object> constantValue, bool isImplicit) :
-            base(semanticModel, node, constantValue: constantValue, isImplicit: isImplicit)
+        public LazyNoneOperation(SemanticModel semanticModel, SyntaxNode node, Optional<object> constantValue, bool isImplicit, ITypeSymbol type = null) :
```

Comment:
>ITypeSymbol type = null [](start = 128, length = 23)

It looks like the ```NoneOperation``` and clonner should be adjusted as well. It would be more robust to make this parameter required.